stdlib/src/commonTest/kotlin/it/unibo/collektive/stdlib/test/TestDistanceTo.kt

```diff
@@ -0,0 +1,66 @@
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+import it.unibo.collektive.stdlib.distanceTo
+import it.unibo.collektive.testing.Environment
+import it.unibo.collektive.testing.mooreGrid
+import kotlin.math.abs
+import kotlin.math.sqrt
+
+/*
+ * Copyright (c) 2024, Danilo Pianini, Nicolas Farabegoli, Elisa Tronetti, and all authors listed in the `build.gradle.kts` and the generated `pom.xml` file.
+ *
+ * This file is part of Collektive, and is distributed under the terms of the Apache License 2.0,
+ * as described in the LICENSE file in this project's repository's top directory.
+ */
+
+class TestDistanceTo : StringSpec({
+    fun Environment<Double>.gradientIsStable(isMoore: Boolean): Boolean = status().all { (id, value) ->
```

Comment:
Maybe this function can be extracted in a utils object for reuse purposes.

Reply:
Not really. It makes assumptions about the specific algorithm